src/main/java/forceitembattle/util/Timer.java

```diff
@@ -150,7 +150,7 @@ public void run() {
                             }
                         } else {
                             if (player.getInventory().contains(ForceItemBattle.getGamemanager().getCurrentMaterial(player))) {
-                                ForceItemBattle.getGamemanager().checkItem(player, ForceItemBattle.getGamemanager().getCurrentMaterial(player), false);
+                                //ForceItemBattle.getGamemanager().checkItem(player, ForceItemBattle.getGamemanager().getCurrentMaterial(player), false);

```

Comment:
What was this used for? Last second item finds?

Reply:
This was the previous method to check if someone found a item. Like this method got called every second and checked in your inventory whether you have the next item or not. But since I changed the whole system, its not needed